src/aavrutiDJun20/BankingSystem.java

```diff
@@ -0,0 +1,86 @@
+package aavrutiDJun20;
+
+public class BankingSystem {
+
+	int user2printBalCount = 0;
+	int currentBalance = 0;
+	int debitCount = 0;
+	int creditCount = 0;
+	int printBalanceCount = 0;
+	static int allUsersDebitCnt;
+	static int allUsersCreditCnt;
+	static int allUsersPrintBalance;
+	
+	int amountCreditedByUser(int userCreditAmount) {
+		if(userCreditAmount>=0) {
+			currentBalance = currentBalance + userCreditAmount;
+			System.out.println("Credit Amount : " + userCreditAmount );
+			allUsersCreditCnt++;
+			return ++creditCount;
+		}
+		else {
+			System.out.println("Please enter credit amount more than 0");
+			return creditCount;
+		}
+	}
+	
+	int amountDebitedByUser(int userDebitAmount) {
+		if(userDebitAmount>=0 && userDebitAmount<=currentBalance) {
+			allUsersDebitCnt++;
+			System.out.println("Debit Amount  : " + userDebitAmount );
+			currentBalance = currentBalance - userDebitAmount;
+			return ++debitCount;
+		}
+		else {
+			System.out.println("Please enter valid debit amount it should not excess current balance.");
+			return debitCount;
+		}
+	}
+	
+	int printBalance(String userName) {
```

Comment:
Return statement not needed for any method because you have created an instance variable for every operation.

Reply:
Done